### DISCUSSION TITLE
Enabling texture dump

### DISCUSSION THREAD

**Discussion by niltok64** [project64/project64#2142] [Support]
I want to make a texture pack for an N64 game, but I don't know how to enable texture dumping on Project64. I checked on the forums but didn't find any concrete answers. I'm using GLideN64. Any help?

**Comment by DerekTurtleRoe** [comment]
@niltok64 Please ask questions about GLideN64 on their dedicated GitHub discussions: https://github.com/gonetz/GLideN64/discussions
